core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-ema-info-display/dot-ema-info-display.component.ts

```diff
@@ -19,7 +20,7 @@ export class DotEmaInfoDisplayComponent {
     protected readonly uveStore = inject(UVEStore);
     protected readonly router = inject(Router);
 
-    protected readonly $options = this.uveStore.$infoDisplayOptions;
+    $options = input<InfoOptions>(undefined, { alias: 'options' });
```

Comment:
Is this the standard we are going to use for `inputs`? @dotCMS/frontend 

`inputs` with `$` as a prefix?